Ginger/GingerCore/Drivers/SeleniumDriver.cs

```diff
@@ -5720,6 +5720,18 @@ private void SwitchWindow(Act act)
                                     break;
                                 }
                             }
+                            if (act.LocateBy == eLocateBy.ByIndex)
+                            {
+                                int getWindowIndex = Int16.Parse(act.LocateValue);
```

Comment:
if locate value is using value expression then it will not work.
Please use locatevalueCaluted